packages/worker/src/utilities/email.ts

```diff
@@ -162,6 +164,42 @@ export async function sendEmail(
       contents: opts?.contents,
     }),
   }
+  if (opts?.attachments) {
+    const attachments = await Promise.all(
+      opts.attachments?.map(async attachment => {
```

Comment:
We don't need to worry for now, but this can cause some kind of DDoS if we don't limit the attachment length